axelrod/tests/unit/test_tournament.py

```diff
@@ -177,44 +303,74 @@ def test_progress_bar_play(self):
         self.assertIsInstance(results, axelrod.ResultSet)
-        self.assertEqual(tournament.progress_bar.total, 15)
-        self.assertEqual(tournament.progress_bar.total,
-                         tournament.progress_bar.n)
-
-        results = tournament.play(progress_bar=True)
+        # Check that progress bar wrote output.
+        stderr_output = err.getvalue()
+        self.assertIn('Playing matches:', stderr_output)
+        self.assertIn('Analysing:', stderr_output)
+        self.assertIn('100%', stderr_output)
+
+        err = io.StringIO()
+        with redirect_stderr(err):
+            results = tournament.play(progress_bar=True)
         self.assertIsInstance(results, axelrod.ResultSet)
-        self.assertEqual(tournament.progress_bar.total, 15)
-        self.assertEqual(tournament.progress_bar.total,
-                         tournament.progress_bar.n)
+        stderr_output = err.getvalue()
+        self.assertIn('Playing matches:', stderr_output)
+        self.assertIn('Analysing:', stderr_output)
+        self.assertIn('100%', stderr_output)
 
         # Test without build results
-        results = tournament.play(progress_bar=True, build_results=False,
-                                  filename=self.filename)
+        err = io.StringIO()
+        with redirect_stderr(err):
+            results = tournament.play(progress_bar=True, build_results=False,
+                                      filename=self.filename)
         self.assertIsNone(results)
+        stderr_output = err.getvalue()
+        self.assertIn('Playing matches:', stderr_output)
+        self.assertNotIn('Analysing:', stderr_output)
+        self.assertIn('100%', stderr_output)
+
         results = axelrod.ResultSetFromFile(self.filename)
         self.assertIsInstance(results, axelrod.ResultSet)
-        self.assertEqual(tournament.progress_bar.total, 15)
-        self.assertEqual(tournament.progress_bar.total,
-                         tournament.progress_bar.n)
 
-    @unittest.skipIf(axelrod.on_windows,
-                     "Parallel processing not supported on Windows")
     def test_progress_bar_play_parallel(self):
         """Test that tournament plays when asking for progress bar for parallel
-        tournament"""
+        tournament and that progress bar is created."""
         tournament = axelrod.Tournament(
             name=self.test_name,
             players=self.players,
             game=self.game,
             turns=axelrod.DEFAULT_TURNS,
             repetitions=self.test_repetitions)
 
-        results = tournament.play(processes=2)
+        # progress_bar = False
+        err = io.StringIO()
+        with redirect_stderr(err):
+            results = tournament.play(progress_bar=False, processes=2)
+        self.assertEqual(err.getvalue(), '')
         self.assertIsInstance(results, axelrod.ResultSet)
 
-        results = tournament.play(progress_bar=True)
+        # progress_bar = True
+        err = io.StringIO()
```

Comment:
Is this setting things so that the progress bar is not output during the tests? (apologies)